.env.example

```diff
@@ -7,6 +7,6 @@ export MADELINE_SMTP_PORT=587
 export MADELINE_SMTP_DOMAIN=example.com
 export MADELINE_SMTP_USERNAME=username
 export MADELINE_SMTP_PASSWORD=xxx
-
+export MADELINE_ERROR_EMAILS_TO=madeline.ops@sassafras.coop
```

Comment:
Should this be committed?

Reply:
Good point. I'll change it to something generic.

Reply:
Thanks.